Cargo.toml

```diff
@@ -16,5 +16,7 @@ categories = [
 publish = false
 
 [dependencies]
+chrono = "0.4.13"
+diesel = { version = "1.4.5", features = ["postgres"] }
```

Comment:
**Q:** Do we need to install `diesiel-cli` from `master`?

Reply:
My current understanding is that this is not possible at the moment, because `diesel_cli` is a binary-only crate :( https://github.com/rust-lang/cargo/issues/4316

Let me try using https://docs.rs/diesel_migrations/1.4.0/diesel_migrations/index.html in a follow up PR. Or just replace this with alembic ;)